logistics/scripts/modules/database/postgres-database-management.psm1

```diff
@@ -7,6 +7,7 @@ $ErrorActionPreference = "Stop"
 
 & "$PSScriptRoot\..\..\..\..\logistics\scripts\modules\load-path-resolver.ps1"
```

Comment:
Since you're already here, would it make sense to go ahead and change these to forward slashes now?

Reply:
I was actually set to start doing this on the next ticket, but then realized while testing on Unix that it actually didn't cause issue...do we still want to do it anyway?